discovery/consul/consul.go

```diff
@@ -24,9 +25,9 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/log"
 	"github.com/prometheus/common/model"
-	"golang.org/x/net/context"
```

Comment:
So personally I don't care, but the current three blocks had a reason: 1) imports from stdlib 2) imports from external packages 3) imports from internal (prometheus/prometheus/...) packages. You merged the second and third group together.